source/application/BioXAS/BioXASImagingAppController.cpp

```diff
@@ -151,12 +145,8 @@ void BioXASImagingAppController::createComponentsPane()
 	mw_->addPane(paneWidget, componentPaneCategoryName_, "Carbon Filter Farm", componentPaneIcon_);
 }
 
-void BioXASImagingAppController::onCurrentScanActionStartedImplementation(AMScanAction *action)
-{
-	Q_UNUSED(action)
-}
-
 void BioXASImagingAppController::onCurrentScanActionFinishedImplementation(AMScanAction *action)
 {
-	Q_UNUSED(action)
+	// perfrom the general actions when scan action finished
+	CLSAppController::onCurrentScanActionFinishedImplementation(action);
```

Comment:
It won't. We checked `userConfiguration_` before using it. I put it here just to make sure we won't forgot to call it in the future.